webapp/channels/src/components/menu/menu.tsx

```diff
@@ -193,7 +194,8 @@ export function Menu(props: Props) {
                 aria-haspopup={true}
                 aria-expanded={isMenuOpen}
                 disabled={props.menuButton?.disabled ?? false}
-                aria-label={props.menuButton?.['aria-label'] ?? ''}
+                aria-label={props.menuButton?.['aria-label']}
```

Comment:
Thanks for changing this. I meant to mention that we shouldn't default these to the empty string since I think that would override the automatic accessible name for the component